src/test/java/com/google/sps/servlets/DBUtilitiesTest.java

```diff
@@ -0,0 +1,99 @@
+package com.google.sps.servlets;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.servlets.DBUtilities;
+import com.google.sps.servlets.Property;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+
+@RunWith(JUnit4.class)
+public final class DBUtilitiesTest {
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  private DatastoreService setDatabaseUp() {
+    DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+    
+    Entity originalInteraction = new Entity(DBUtilities.INTERACTION_TABLE);
+    originalInteraction.setProperty(Property.CORRELATOR, "Person1");
+    originalInteraction.setProperty(Property.FIND_NEAREST_LOCATION, false);
+    originalInteraction.setProperty(Property.GRANTS_LOCATION, false);
+    originalInteraction.setProperty(Property.INTERACTS_WITH_MAP, false);
+    originalInteraction.setProperty(Property.SKIP_TO_CONTENT, false);
+    originalInteraction.setProperty(Property.RETURN_TO_AD, false);
+
+    dataStore.put(originalInteraction);
+
+    return dataStore;
+  }
+
+  @Test
+  public void testIfOnlyOneEntryMade() {
```

Comment:
I don't think I see what this is testing.  This would pass even if you never called anything DBUtils

Reply:
the idea for this test is to make sure the setToTrue function overrides the entry and doesn't end up making a duplicate entry with an updated field.

Reply:
Ahh gotcha makes sense.  Maybe just change the name of the test a bit to specify that.

`testSetToTrueDoesntCreateNewEntry` or something else a bit more specific.

For test names especially, since you never actually call them anywhere, theyre fine if theyre super long